website/client/src/components/shops/quests/questDialogContent.vue

```diff
@@ -22,36 +28,80 @@
 <style lang="scss" scoped>
   @import '~@/assets/scss/colors.scss';
 
+  h3 {
+    color: $gray-10;
+    margin-bottom: 0.25rem;
+  }
+
   .quest-image {
     margin: 0 auto;
     margin-bottom: 1em;
     margin-top: 1.5em;
   }
 
   .text {
-    margin-bottom: 8px;
+    margin-bottom: 1rem;
     overflow-y: scroll;
     text-overflow: ellipsis;
   }
 
+  .leader-label {
+    font-size: 14px;
+    font-weight: bold;
+    line-height: 1.71;
+    color: $gray-50;
+    text-align: center;
+    margin-bottom: 0.5rem;
+
+    ::v-deep .user-label {
+      font-size: 14px;
+    }
+  }
+
   .questInfo {
-    width: 70%;
+    width: 160px;
     margin: 0 auto;
-    margin-bottom: 10px;
+
+    display: flex;
+    justify-content: center;
   }
 </style>
 
 <script>
 import QuestInfo from './questInfo.vue';
+import UserLabel from '../../userLabel';
 
 export default {
   components: {
+    UserLabel,
     QuestInfo,
   },
+  data () {
+    return {
+      leader: null,
+    };
+  },
   props: {
     item: {
       type: Object,
     },
+    group: {
+      type: Object,
+    },
+  },
+  async created () {
+    if (this.group && this.group.quest && this.group.quest.active) {
+      try {
+        const fetchMemberResult = await this.$store.dispatch('members:fetchMember', {
+          memberId: this.group.quest.leader,
+        });
+
+        // this should be handled in fetchMember instead
```

Comment:
a TODO? something else we should ticket I guess?

Reply:
I meant with that - that the `fetchMember` return needs to be unpacked `data.data` first before its used - I can create a refactor issue for that

Reply:
or change that method so that all usings don't need to go for `.data.data` all the time